preprocessing/berlin/compile-berlin-geojson.js

```diff
@@ -11,6 +11,17 @@ var opening_hours = require('opening_hours');
 var INPUT_FILE = "./preprocessing/berlin/raw/markets-berlin.json";
 var OUTPUT_FILE = "./cities/berlin.json";
 
+// Sort input by title
+fs.readFile(INPUT_FILE, 'utf8', function (error, data) {
+	if (error) {
+		throw error;
+	}
+	var featureCollection = JSON.parse(data);
+	var outputData = featureCollection;
+  outputData.features = outputData.features.sort(function(a,b){return a.properties.title > b.properties.title ? 1 : -1;});
```

Comment:
Formatting: Please adjust the indentation and add some whitespace within the function.